app/src/main/java/com/example/mytestproject/weatherData/WeatherDataFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.mytestproject.weatherData
+
+import android.graphics.drawable.AnimationDrawable
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.Observer
+import com.example.mytestproject.R
+import com.squareup.picasso.Picasso
+import kotlinx.android.synthetic.main.fragment_first.*
+
+class WeatherDataFragment : Fragment() {
+
+    private val weatherDataViewModel: WeatherDataViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_first, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val animationDrawable = fragment_1_background.background as AnimationDrawable
+        animationDrawable.setEnterFadeDuration(4000)
+        animationDrawable.setExitFadeDuration(4000)
+        animationDrawable.start()
+
+        setupWeatherData()
+
+    }
+
+    private fun setupWeatherData() {
+        weatherDataViewModel.getWeatherData("Moscow", 1, "M")
+        weatherDataViewModel.weatherDataApi.observe(viewLifecycleOwner, Observer {
+            textTemperature.text = it.data[0].temp.toString()
```

Comment:
Screen is very simple. Let's use DataBinding. 
Custom BindingAdapter could be used for image loading.